packages/babel-runtime/package.json

```diff
@@ -6,7 +6,11 @@
   "publishConfig": {
     "access": "public"
   },
-  "repository": "https://github.com/babel/babel/tree/master/packages/babel-runtime",
+  "repository": {
```

Comment:
On the `repository`, NPM offers an extra [`directory` field](https://docs.npmjs.com/files/package.json#repository) for monorepo:

```json
"repository": {
  "type" : "git",
  "url" : "https://github.com/babel/babel.git",
  "directory": "packages/babel-runtime"
}
```
To be consistent across different packages in this repo, I think we had better to replace all the repository fields.

Reply:
Thanks for pointing that out! I will make the update here.